tree/dataframe/test/dataframe_utils.cxx

```diff
@@ -233,3 +235,21 @@ TEST(RDataFrameUtils, TypeName2TypeID)
    // TODO(jblomer): Ideally, we would want the next one not to throw an exception
    EXPECT_THROW(RDFInt::TypeName2TypeID("std::vector<std::vector<float>>"), std::runtime_error);
 }
+
+TEST(RDataFrameUtils, JittedFilters)
```

Comment:
we need a more descriptive name for the test case, and it should go e.g. in `dataframe_interface.cxx` or `dataframe_regression.cxx`. `dataframe_utils.cxx` is meant to test code in `InterfaceUtils.hxx` or `Utils.hxx`.

Reply:
Renamed to `RegexWithFriendsInJittedFilters` in `dataframe_interface.cxx`